brianlong/www.validators.app
# 186628632 update smtp credentials

#### What's this PR do?
- Updates smtp credentials and updates email sender

#### How should this be manually tested?
- Add validator to your watchlist (star icon)
- change its commission via console
- confirm that you received notification on your email

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186628632)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
